src/components/prescriptions/PrescriptionComponent.vue

```diff
@@ -119,4 +119,79 @@
             @include text-header6($font-weight-regular);
         }
     }
+
+    @media (min-width: $zero-width) and (max-width: $tablet-width) {
```

Comment:
Not sure about min width with 0 is necessary? If range is 0 < x<= e.g. 750px its not required to declare min-with 0px where containers does not negative width

Reply:
wiesz co, ja to wykonalam ze wzgledu na problem o ktorym pisałam - dla 1024 style dzialaly, dla 1370 działaly a jak doszlo do ipada pobieralo style zamiast do 768 to do 1370 - na stackoverflow znalazlam takie rozwiazanie i zadziałało i dobrze czyta style